rebac-admin-backend/v1/interfaces/entitlements.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2024 Canonical Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package interfaces
+
+import (
+	"context"
+
+	"github.com/canonical/identity-platform-admin-ui/rebac-admin-backend/v1/resources"
+)
+
+type EntitlementsService interface {
+	// ListEntitlements returns the list of entitlements in JSON format.
+	ListEntitlements(ctx context.Context, params *resources.GetEntitlementsParams) ([]resources.EntityEntitlement, error)
+
+	// RawEntitlements returns the list of entitlements as raw text.
+	RawEntitlements(ctx context.Context) (*string, error)
```

Comment:
Since this is a primitive return type, we can just use the value type.

Reply:
I made it a pointer so it can be nillable in case of an error is present, but I think it's safe to make it concrete and be consistent

Reply:
Done with the new commit, thanks!